centos_7_py3/cloudify-wagon-builder.sh

```diff
@@ -14,10 +14,10 @@ echo "manylinux1_compatible = False" > "/env/bin/_manylinux.py"
 if test -f ${CONSTRAINTS_FILE}
 then
     echo "## $CONSTRAINTS_FILE exist"
-    wagon create -r ${REQUIRMENTS_FILE} -v -f -a '--no-cache-dir -c '${CONSTRAINTS_FILE}'' .
+    wagon create -r ${REQUIRMENTS_FILE} -a '--no-cache-dir -c --build-tag "centos-Core" -r ${REQUIREMENTS_FILE} -v -f .' .
```

Comment:
you are missing the constraints file.

Reply:
wagon create  --build-tag "centos-Core" -r ${REQUIREMENTS_FILE} -v -f -a '--no-cache-dir -c '${CONSTRAINTS_FILE}'' .